bumble/transport/common.py

```diff
@@ -63,6 +63,8 @@ def on_packet(self, packet: bytes) -> None:
 
 
 class TransportSource(Protocol):
+    terminated: asyncio.Future[None]
```

Comment:
I got error with it
```
bumble/transport/common.py:390: error: Incompatible types in assignment (expression has type "PumpedPacketSource", base class "Transport" defined the type as "TransportSource")  [assignment]
bumble/transport/common.py:390: note: Following member(s) of "PumpedPacketSource" have conflicts:
bumble/transport/common.py:390: note:     terminated: expected "Awaitable[Any]", got "Future[None]"

```

Reply:
Ah, yes, I forgot, `terminated` here is `Awaitable` from the point of view of the user of the transport, but need to be a `Future` from the point of view of the implementation, because we want to be able to call `set_result` and `set_exception` on it. So `Future[None]` is the right type.